spec/models/ability_spec.rb

```diff
@@ -38,6 +38,7 @@
       it { is_expected.not_to be_able_to(:create, client) }
       it { is_expected.not_to be_able_to(:update, client) }
       it { is_expected.not_to be_able_to(:destroy, client) }
+      it { is_expected.not_to be_able_to(:transfer, client) }
```

Comment:
Again, please clarify whether permissions for custom actions need to be specifically assigned, and test accordingly.